src/plugin/file-qml-plugin/docviewerutils.cpp

```diff
@@ -76,6 +76,8 @@ QString DocviewerUtils::getXdgDocumentsLocation()
 
 QString DocviewerUtils::buildDestinationPath(const QString &destinationDir, const QString &sourcePath)
 {
+    Q_UNUSED( destinationDir )
```

Comment:
Please keep the coding style consistent: `Q_UNUSED(destinationDir);`
(also note the semicolon at the end)